specifyweb/frontend/js_src/lib/components/BatchEdit/index.tsx

```diff
@@ -170,100 +158,114 @@ const hasHierarchyBaseTable = (queryFieldSpec: QueryFieldSpec) =>
   );
 
 const containsTreeTableOrSpecificRank = (queryFieldSpec: QueryFieldSpec) =>
-  isTreeTable(queryFieldSpec.baseTable.name) ||
-  (typeof queryFieldSpec.treeRank === 'string' &&
-    queryFieldSpec.treeRank !== '-any');
+  isTreeTable(queryFieldSpec.baseTable.name);
 
 const filters = [containsFaultyNestedToMany, containsSystemTables];
 
-const getTreeDefFromName = (
-  rankName: string,
-  treeDefItems: RA<SerializedResource<FilterTablesByEndsWith<'TreeDefItem'>>>
-) =>
-  defined(
-    treeDefItems.find(
-      (treeRank) => treeRank.name.toLowerCase() === rankName.toLowerCase()
-    )
-  );
+/*
+ * Const getTreeDefFromName = (
+ *   rankName: string,
+ *   treeDefItems: RA<SerializedResource<FilterTablesByEndsWith<'TreeDefItem'>>>
+ * ) =>
+ *   defined(
+ *     treeDefItems.find(
+ *       (treeRank) => treeRank.name.toLowerCase() === rankName.toLowerCase()
+ *     )
+ *   );
+ */
```

Comment:
(optional)

Are these comments being kept because they will be uncommented when we add support for trees in Batch Edit (https://github.com/specify/specify7/issues/6127)? 

If so, could there either be a simple comment above each of these either explaining when it should be added back (and thus not removed) (e.g., `Uncomment when #6127 is implemented`), or removed and a single link to this PR (and/or #6127) placed in a spot you think makes the most sense (like maybe with the other related [TODO comment](https://github.com/specify/specify7/blob/12c936161a5f58cc8aab8fc452ab3ff3ccbb7008/specifyweb/frontend/js_src/lib/components/BatchEdit/index.tsx#L84)?)

Reply:
Yeah, they will be uncommented in 6127. In fact, most of this file has been split into 4 smaller files in #6127 so all this code doesn't even exist here in that PR